media/css/eoy.css

```diff
@@ -0,0 +1,367 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@font-face {
+  font-family: system-ui;
+  font-style: normal;
+  font-weight: 300;
+  src: local(".SFNS-Light"), local(".SFNSText-Light"), local("Segoe UI Light"), local("Ubuntu Light");
+}
+
+@font-face {
+  font-family: system-ui;
+  font-style: italic;
+  font-weight: 300;
+  src: local(".SFNS-LightItalic"), local(".SFNSText-LightItalic"), local("Segoe UI Light Italic"), local("Ubuntu Light Italic");
+}
+
+@font-face {
+  font-family: system-ui;
+  font-style: normal;
+  font-weight: 400;
+  src: local(".SFNS-Regular"), local(".SFNSText-Regular"), local("Segoe UI"), local("Ubuntu");
+}
+
+@font-face {
+  font-family: system-ui;
+  font-style: italic;
+  font-weight: 400;
+  src: local(".SFNS-Italic"), local(".SFNSText-Italic"), local("Segoe UI Italic"), local("Ubuntu Italic");
+}
+
+@font-face {
+  font-family: system-ui;
+  font-style: normal;
+  font-weight: 500;
+  src: local(".SFNS-Medium"), local(".SFNSText-Medium"), local("Segoe UI Semibold"), local("Ubuntu Medium");
+}
+
+@font-face {
+  font-family: system-ui;
+  font-style: italic;
+  font-weight: 500;
+  src: local(".SFNS-MediumItalic"), local(".SFNSText-MediumItalic"), local("Segoe UI Semibold Italic"), local("Ubuntu Medium Italic");
+}
+
+@font-face {
+  font-family: system-ui;
+  font-style: normal;
+  font-weight: 700;
+  src: local(".SFNS-Bold"), local(".SFNSText-Bold"), local("Segoe UI Bold"), local("Ubuntu Bold");
+}
+
+@font-face {
+  font-family: system-ui;
+  font-style: italic;
+  font-weight: 700;
+  src: local(".SFNS-BoldItalic"), local(".SFNSText-BoldItalic"), local("Segoe UI Bold Italic"), local("Ubuntu Bold Italic");
+}
+
+:root {
+  --bg: #f9f9fa;
+  --text: #333;
+  --shadow: 0 4px 16px rgba(0, 0, 0, 0.1);
+  --accent-text: #15427C;
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --bg: #2F2F33;
+    --text: #F9F9FA;
+    --shadow: 0 4px 16px rgba(0, 0, 0, 0.3);
+    --accent-text: #DDEEFE;
+  }
+}
+
+html {
+  font-family: system-ui, Open Sans, sans-serif;
+  font-size: 1rem;
+}
+
+body {
+  background-color: #B7E0FC;
+  background-image: url('/media/img/thunderbird/backgrounds/forest-bg-light.jpg');
+  background-position: top left;
+  background-size: auto 400px;
+  background-repeat: repeat-x;
+  color: var(--text);
+  position: relative;
+  margin: 0;
+  padding: 0;
+  min-height: 100vh;
+  max-width: 100%;
+  line-height: 1.6;
+}
+
+@media (prefers-color-scheme: dark) {
+  body {
+    background: #204571;
+    background-position: top left;
+    background-size: auto 400px;
+    background-repeat: repeat-x;
```

Comment:
no need to repeat the same values for dark mode, these should be maintained from the previous body style.